packages/RatPack.package/RPTestApp.class/class/routeNamingStrategies.st

```diff
@@ -0,0 +1,4 @@
+as yet unclassified
+routeNamingStrategies 
+
+	^ OrderedCollection newFrom: { RPTestRouteNamingStrategy new. }.
```

Comment:
@tom95 suggested `^ { RPTestRouteNamingStrategy new. }.`

Reply:
It has to be ordered (if you have more than one strategy different execution order leads to different results)

Reply:
`^ { RPTestRouteNamingStrategy new. } asOrderedCollection` then, sorry